packages/test/test-end-to-end-tests/src/test/gc/gcInactiveNodes.spec.ts

```diff
@@ -70,7 +64,6 @@ describeNoCompat("GC inactive nodes tests", (getTestObjectProvider) => {
 		const { throwOnInactiveLoad } = params;
 		return {
 			runtimeOptions: {
-				summaryOptions: { summaryConfigOverrides: { state: "disabled" } },
```

Comment:
Why is this removed?

Reply:
A lot of tests are creating summarizer containers, and `Summarizer` objects won't exist when you have `state: "disabled"`. This is a problem when you try to get default data objects from the summarizer container as it'll just hit assert `0x5bf /* Summarizer object is undefined in a summarizer client */`